backend/api/authorization/serializers.py

```diff
@@ -0,0 +1,11 @@
+from apps.users.models import User
+from rest_framework import serializers
+
+
+class UserLoginSerializer(serializers.ModelSerializer):
+    email = serializers.CharField()
```

Comment:
Declaring email as serializers.CharField() may lead to issues if User model uses EmailField. Use serializers.EmailField.
```
email = serializers.EmailField()
```